packages/relay/src/routes/counter.ts

```diff
@@ -0,0 +1,55 @@
+import { DB } from 'anondb/node'
+import { Express } from 'express'
+import { errorHandler } from '../middleware'
+import { UnirepSocialSynchronizer } from '../synchornizer'
+
+export default (
+    app: Express,
+    db: DB,
+    synchronizer: UnirepSocialSynchronizer
+) => {
+    app.get(
+        '/api/counter',
+        errorHandler(async (req, res, next) => {
+            await fetchActions(req, res, db)
+        })
+    )
+}
+
+/**
+ * fetch the number of actions of three epoch keys which derived from the identity
+ *
+ * @param req express request
+ * @param res express response
+ * @param db SQLiteConnector
+ */
+async function fetchActions(req, res, db: DB) {
+    try {
+        const epks = req.query.epks ? req.query.epks.split('_') : undefined
+        // each user has 3 epoch keys during the epoch
+        if (epks.length != 3) {
```

Comment:
Should we make this become env variable to avoid hard code

Reply:
I set an env variable `EPOCHKEYS_AMOUNT` on `config.ts`